### DISCUSSION TITLE
Asset folder

### DISCUSSION THREAD

**Discussion by Penfret** [requarks/wiki#6230] [Help / Questions] [answered]
Hello,
Is it possible to delete an asset folder ? We can only see the possibility to create a folder and delete asset files, but not folders.
Are we missing something or is it not supported?
Thanks,
Anne-Cécile

**Comment by NGPixel** [comment, marked as answer]
This is not possible at the moment. Coming in a future release.

**Comment by Penfret** [comment]
Ok, do you have any idea of the delay before the future release ?
Thanks ;)
